webservices/soap/client/src/main/java/com/epam/soap/Main.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.soap;
+
+import com.epam.soap.service.FactorialService;
+import com.epam.soap.service.FactorialService_Service;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class Main {
+
+    public static void main(String[] args) throws MalformedURLException {
+        URL wsdl = new URL("http://127.0.0.1:8081/FactorialService?wsdl");
```

Comment:
As we discussed, this could be solved with embedded Camel, for example. But that's ok for now.